packages/core/src/vars.css

```diff
@@ -169,6 +169,12 @@
 	--spinner-color-blue: var(--ts-color-blue);
 	--spinner-color-mono: var(--ts-color-slate-lightest);
 	--spinner-color-white: var(--ts-color-white);
+
+	/* Radio button........................................................ */
+
+	--radio-border-selected: 7px;
```

Comment:
You should use one of the variables. I think ts-unit-quarter.

Reply:
I made it 7px to make it identical to _ts-ui_. If it's not required, surely I'll use ts-unit variable

Reply:
Please ask @mihneaz or Vlad to get a confirmation about the size, but it should be at least 
 similar to ts-ui if not identical 👍